src/Kdyby/Events/DI/EventsExtension.php

```diff
@@ -71,7 +71,8 @@ public function loadConfiguration()
 		$config = $this->getConfig($this->defaults);
 
 		$userConfig = $this->getConfig();
-		if (!isset($userConfig['debugger']) && !$config['debugger']) {
+		$isCli = in_array(php_sapi_name(), ['cli', 'phpdbg'], TRUE);
+		if ($isCli || (!isset($userConfig['debugger']) && !$config['debugger'])) {
```

Comment:
It may seem stupid to have the panel enabled in cli, but this should be controlled by the programmer. For example I have an extension, that forcefully renders the the panel into a html file.

I would be more inclined to change the default value to `off`, so you have to explicitly enable the panel.
It's only meant for debugging and it's quite slow, so having it disabled as default might be a benefit.